mathesar/templates/mathesar/base.html

```diff
@@ -1,18 +1,25 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="UTF-8" />
-  <title>Mathesar - {% block title %}{% endblock %}</title>
-  {% block styles %}
-  {% endblock %}
-</head>
-<body>
-  <div>
-    <a href="/">Mathesar</a>
-  </div>
-  {% block body %}
-  {% endblock %}
-  {% block scripts %}
-  {% endblock %}
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Mathesar - {% block title %}{% endblock %}</title>
+    {% block styles %}
+    {% endblock %}
+  </head>
+  <body>
+    <div>
+      <a href="/">Mathesar</a>
+    </div>
+    {% block body %}
+    {% endblock %}
+    {% block scripts %}
+    {% endblock %}
+
+    <!-- For development with hot module replacement -->
+    <!-- TODO: Control mode with an env variable -->
+    <script type="module" src="http://localhost:3000/@vite/client"></script>
```

Comment:
I don't think we should hardcode this, can we use a Django setting to specify the URL here?

Reply:
I've used a context processor to be able to get the url in templates. Based on the quick reading I did, this seemed like the optimal way, but let me know if this is not okay.

Reply:
That works.